ecosystem/lava-sdk/src/stateTracker/updaters/updater.ts

```diff
@@ -1,3 +1,3 @@
 export interface Updater {
-  update(): Promise<any>;
+  update(virtualEpoch: number): Promise<any>;
```

Comment:
i don't want us to update the updater interface to have the virtual epoch, no need for this in the go implementation, no need for this here